unsupported/test/CMakeLists.txt

```diff
@@ -319,17 +319,18 @@ if (EIGEN_TEST_HIP)
 
 	set(EIGEN_ADD_TEST_FILENAME_EXTENSION  "cu")
 
-	# ei_add_test(cxx11_tensor_complex_hip)
-	# ei_add_test(cxx11_tensor_complex_cwise_ops_hip)
-	ei_add_test(cxx11_tensor_reduction_hip)
-	ei_add_test(cxx11_tensor_argmax_hip)
-	ei_add_test(cxx11_tensor_cast_float16_hip)
-	ei_add_test(cxx11_tensor_scan_hip)
-	ei_add_test(cxx11_tensor_device_hip)
-	ei_add_test(cxx11_tensor_hip)
-	ei_add_test(cxx11_tensor_contract_hip)
-	ei_add_test(cxx11_tensor_of_float16_hip)
-	ei_add_test(cxx11_tensor_random_hip)
+	# ei_add_test(cxx11_tensor_complex_gpu)
```

Comment:
we probably want to add comment that these tests aren't supported in HIP yet.

Reply:
yup...will add those comments in and update PR later tonight / tomorrow